config/vufind/SierraRest.ini

```diff
@@ -113,16 +108,32 @@ defaultPickUpLocation = ""
 ; By default item holds are enabled. Uncomment this setting to disable item holds.
 ;enableItemHolds = false
 
-; This setting specifies which item codes disable item level holds
-item_hold_excluded_item_codes = e
-
-; This setting specifies which bib levels allow title level holds
-title_hold_bib_levels = a:b:m:d
+; If a colon separated list of item statuses is provided here, only matching items
+; will show hold links.  Skip this setting to allow all statuses.
+; Note that VuFind-style statuses are used here.
+;valid_hold_statuses = 'On Shelf:Charged'
 
-; By default a title hold can be placed even when there are no items. Uncomment this
-; to prevent holds if no items exist. If request groups are enabled, item existence
-; is checked only in the selected request group.
-;checkItemsExist = true
```

Comment:
I notice that you've completely removed this setting -- was this something that was never fully implemented? I don't see any code related to the setting in SierraRest.php, but there is a property with a corresponding name that doesn't seem to be used for anything. Should that be removed too?

Reply:
It was never implemented at all, but I forgot to remove the property. Now done.